src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Venues/VenuesDataManagementController.cs

```diff
@@ -203,5 +204,33 @@ public IActionResult Formatting()
         {
             return View();
         }
+
+        [HttpPost("resolve/delete")]
```

Comment:
Would be good to have this route as `resolve/{rowNumber}/delete` so it aligns with the error fix pages.

Reply:
done